Containerfile

```diff
@@ -54,14 +54,16 @@ RUN cargo build \
     --bin shuttle-next -F next
 
 
+####### Helper step
+
+FROM docker.io/library/debian:bookworm-20230904-slim AS bookworm-20230904-slim-plus
+RUN apt update && apt install -y curl ca-certificates; rm -rf /var/lib/apt/lists/*
```

Comment:
Why are we doing this for all projects and not just `auth`?

Reply:
Common for auth, gw, and prov was that they can run in the slim image only if they have ca-certs and/or curl. Therefore, this middle step was made.

Reply:
Ohh, I see 👍